sysvabi64/sysvabi64.rst

```diff
@@ -1592,6 +1592,157 @@ IFUNC requirements for dynamic linkers
+    +-------------------+----------+
+    | FEAT_FP           | 1U << 9  |
+    +-------------------+----------+
+    | FEAT_SIMD         | 1U << 10 |
+    +-------------------+----------+
+    | FEAT_CRC          | 1U << 11 |
+    +-------------------+----------+
+    | FEAT_CSSC         | 1U << 12 |
+    +-------------------+----------+
+    | FEAT_SHA2         | 1U << 13 |
+    +-------------------+----------+
+    | FEAT_SHA3         | 1U << 14 |
+    +-------------------+----------+
+    | FEAT_PMULL        | 1U << 16 |
+    +-------------------+----------+
+    | FEAT_FP16         | 1U << 17 |
+    +-------------------+----------+
+    | FEAT_DIT          | 1U << 18 |
+    +-------------------+----------+
+    | FEAT_DPB          | 1U << 19 |
+    +-------------------+----------+
+    | FEAT_DPB2         | 1U << 20 |
+    +-------------------+----------+
+    | FEAT_JSCVT        | 1U << 21 |
+    +-------------------+----------+
+    | FEAT_FCMA         | 1U << 22 |
+    +-------------------+----------+
+    | FEAT_RCPC         | 1U << 23 |
+    +-------------------+----------+
+    | FEAT_RCPC2        | 1U << 24 |
+    +-------------------+----------+
+    | FEAT_FRINTTS      | 1U << 25 |
+    +-------------------+----------+
+    | FEAT_I8MM         | 1U << 27 |
+    +-------------------+----------+
+    | FEAT_BF16         | 1U << 28 |
+    +-------------------+----------+
+    | FEAT_SVE          | 1U << 31 |
+    +-------------------+----------+
+    | FEAT_SVE_F32MM    | 1U << 35 |
+    +-------------------+----------+
+    | FEAT_SVE_F64MM    | 1U << 36 |
+    +-------------------+----------+
+    | FEAT_SVE2         | 1U << 37 |
+    +-------------------+----------+
+    | FEAT_SVE_PMULL128 | 1U << 39 |
+    +-------------------+----------+
+    | FEAT_SVE_BITPERM  | 1U << 40 |
+    +-------------------+----------+
+    | FEAT_SVE_SHA3     | 1U << 41 |
+    +-------------------+----------+
+    | FEAT_SVE_SM4      | 1U << 42 |
+    +-------------------+----------+
+    | FEAT_SME          | 1U << 43 |
+    +-------------------+----------+
+    | FEAT_MEMTAG2      | 1U << 45 |
+    +-------------------+----------+
+    | FEAT_SB           | 1U << 47 |
+    +-------------------+----------+
+    | FEAT_SSBS2        | 1U << 50 |
+    +-------------------+----------+
+    | FEAT_BTI          | 1U << 51 |
+    +-------------------+----------+
+    | FEAT_WFXT         | 1U << 55 |
+    +-------------------+----------+
+    | FEAT_SME_F64      | 1U << 56 |
+    +-------------------+----------+
+    | FEAT_SME_I64      | 1U << 57 |
+    +-------------------+----------+
+    | FEAT_SME2         | 1U << 58 |
+    +-------------------+----------+
+    | FEAT_RCPC3        | 1U << 59 |
+    +-------------------+----------+
+    | FEAT_MOPS         | 1U << 60 |
+    +-------------------+----------+
+
+Implementing FMV using ``__aarch64_cpu_features`` is not required.
+Accessing this variable from outside a FMV resolver function is not
```

Comment:
The `Accessing this variable from outside a FMV resolver function is not well defined.` is in contradiction with the note below. I think we could reword to say:
```
Accessing ``__aarch64_cpu_features`` is reserved for the compiler generated code or the runtime library.
```